csharp/docfx/docfx.json

```diff
@@ -58,7 +58,7 @@
       "_appFaviconPath": "images/favicon.ico",
       "_appLogoPath": "images/logo.png",
       "_appLogoUrl": "https://doc.zeroc.com/api/ice/main/csharp",
-      "_appFooter": "© 2024 ZeroC",
```

Comment:
I generated the docs to see how it would be rendered, and noticed that the copyright years as old.
So I just went ahead and updated it.